httpd-modules.conf

```diff
@@ -5,3 +5,19 @@ LoadModule dir_module modules/mod_dir.so
 LoadModule authz_core_module modules/mod_authz_core.so
 LoadModule unixd_module modules/mod_unixd.so
 LoadModule mpm_event_module modules/mod_mpm_event.so
+<IfModule !wsgi_module>
```

Comment:
If you'd like to run 2 httpd instances, you need two different configurations.

Also, are all these items really required?

Reply:
In this PR,  for ironic, we only have one httpd instance for WSGI and also for IPA to download images. 
All the additional modules are needed for TLS, WSGI and basic authentication on Apache server side.

Reply:
Mmm, but you clearly have httpd starting in runironic-api.sh...

Reply:
Also cannot this configuration go under <VirtualHost *:6385> in the ironic configuration?

Reply:
Yes, I agree that it is not necessary to have the conditional expression here. I have updated it. Thanks